pkgs/applications/editors/gnome-builder/default.nix

```diff
@@ -115,8 +106,6 @@ stdenv.mkDerivation rec {
     "-Dnetwork_tests=false"
   ];
 
-  # Some tests fail due to being unable to find the Vte typelib, and I don't
-  # understand why. Somebody should look into fixing this.
   doCheck = true;
```

Comment:
The test is enabled in https://github.com/NixOS/nixpkgs/commit/d206f597b6c87e24bfae30ab5e14e2ee7e0da5cf. Is the comment still applies? I think all tests passes for me.

Reply:
Probably not.